test/integration/version_upgrade_test.go

```diff
@@ -197,19 +197,19 @@ func TestStoppedBinaryUpgrade(t *testing.T) {
 		t.Errorf("failed to stop cluster: %s: %v", rr.Command(), err)
 	}
 
-	args = append([]string{"start", "-p", profile, "--memory=2200", "--alsologtostderr", "-v=1"}, StartArgs()...)
-	rr, err = Run(t, exec.CommandContext(ctx, Target(), args...))
-	if err != nil {
-		t.Fatalf("upgrade from %s to HEAD failed: %s: %v", desiredLegacyVersion, rr.Command(), err)
-	}
-
 	t.Run("MinikubeLogs", func(t *testing.T) {
```

Comment:
Control plane node must be running for minikube logs. All other environments get upset about moving the start.